### DISCUSSION TITLE
how can we bind onChange Event with input type field in nova

### DISCUSSION THREAD

**Discussion by patrick-paudel-arvig** [laravel/nova-issues#3510] [Q&A] [answered]
I want to associate change event with input field and then extract some text from that then create a new row of repeatable fields.

**Comment by crynobone** [comment, marked as answer]
You can use `Nova.$on('email-change', (value) => ... )`

**Reply by esameisa** [reply]
where i write this code ?

**Reply by juyal-ahmed** [reply]
Does it really works? I tried it on mounted and also on some other methods and found not working and finally I had to use javascript raw event listeners to get those changes by DOM ID!!!
